json/circe/src/test/scala/sttp/client4/circe/CirceTests.scala

```diff
@@ -107,11 +107,13 @@ class CirceTests extends AnyFlatSpec with Matchers with EitherValues {
     import sttp.model.Uri
 
     val jObject: JsonObject = JsonObject(("location", "hometown".asJson), ("bio", "Scala programmer".asJson))
-    val result = basicRequest.get(Uri("http://example.org")).body(jObject).body.show
+    val request = basicRequest.get(Uri("http://example.org")).body(jObject)
 
-    val expectedResult = "string: {\"location\":\"hometown\",\"bio\":\"Scala programmer\"}"
+    val expectedBody = "string: {\"location\":\"hometown\",\"bio\":\"Scala programmer\"}"
+    val expectedContentType = Some("application/json; charset=utf-8")
```

Comment:
application/json is always utf-8, so there should be no charset=, maybe that's why the tests are failing.

Reply:
I checked it and in println we have "application/json; charset=utf-8", but sbt has problem with matching contentType to proper type